models/authors.js

```diff
@@ -1,9 +1,8 @@
 const Authors = (connection, Sequelize) => {
   return connection.define('authors', {
     id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
-    nameFirst: { type: Sequelize.STRING, allowNull: false },
-    nameLast: { type: Sequelize.STRING, allowNull: false },
-  })
+    name: { type: Sequelize.STRING },
```

Comment:
_Nit_ not sure why you changed this. Your program wont run without nameFirst since thats whats declared in your migrations.